tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -219,12 +219,8 @@ def convert_onnx_models_to_ort():
     if args.use_nnapi and 'NnapiExecutionProvider' not in ort.get_available_providers():
         raise ValueError('The NNAPI Execution Provider was not included in this build of ONNX Runtime.')
 
-    if args.use_coreml:
-        if not is_macOS():
```

Comment:
>is_macOS [](http://example.com/codeflow?start=15&length=8)

can we remove the is_macOS() definition as it's no longer used?

Reply:
Yes, forgot to remove that. Thanks for the catch.